runtime/queries/openscad/highlights.scm

```diff
@@ -0,0 +1,60 @@
+(number) @constant.numeric
+(string) @string
+(boolean) @constant.builtin.boolean
+(include_path) @string.special.path
+
+(function_call function: (identifier) @function)
```

Comment:
similar to the above capture, could we add

```
(function_declaration name: (identifier) @function)
```

which marks function names in declarations as functions. Other languages have similar rules

```scad
function flatten(l) = [for (a = l) for (b = a) b];
//         ^ function
```

And along those lines:

```
(parameters_declaration (identifier) @variable.parameter)
```

anywhere above that `(identifier) @variable)` line (so it takes precedence) which would be

```scad
function flatten(l) = [for (a = l) for (b = a) b];
//               ^ variable.parameter
```

Reply:
works great! thanks